error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/CollectionRules.java

```diff
@@ -404,6 +405,19 @@ Optional<T> after(Queue<T> queue) {
     }
   }
 
+  /** Prefer calling {@link Collection#forEach} ()} over more contrived alternatives. */
```

Comment:
```suggestion
  /** Prefer {@link Collection#forEach(Consumer)} over more contrived alternatives. */
```
(I see we use the extra word "calling" in some places in this class, but elsewhere we don't. Something to unify, perhaps in an automated fashion :smile:.